test-suite/output/Cases.out

```diff
@@ -102,11 +102,10 @@ Arguments lem3 k
   n, n0 := match x + 0 with
            | 0 | S _ => 0
            end : nat
-  e,
-  e0 := match x + 0 as y return (y = y) with
-        | 0 => eq_refl
-        | S n => eq_refl
-        end : x + 0 = x + 0
+  e, e0 := match x + 0 as y return (y = y) with
+           | 0 => eq_refl
+           | S n => eq_refl
+           end : x + 0 = x + 0
```

Comment:
Could we get the `match` on a new line so that the branches don't get that indented? eg
```suggestion
  e, e0 :=
    match x + 0 as y return (y = y) with
    | 0 => eq_refl
    | S n => eq_refl
    end : x + 0 = x + 0
```

Reply:
One would need to give a priority on when to break between different nested boxes and I'm unsure to know how to control that :(

Reply:
I would have expected an enclosing box to be enough:, like
"@[<hov 2> @[<hov 0>comma separated variables@] :=@ body@]"
(OCaml format syntax).

Reply:
This is already what we have I think (in `Pp` syntax):
```ocaml
  let pids = hov 0 (prlist_with_sep pr_comma (fun id -> pr_id id.binder_name) ids) in
  let pt = pr_ltype_env env sigma typ in
  let ptyp = (str" : " ++ pt) in
  hov 2 (pids ++ pbody ++ ptyp)
```

Reply:
But in pbody I read `str" := " ++ pb` whereas I would expect `str" :=" ++ spc () ++ pb`

Reply:
Sorry, I missed the break after `:=` in your code. It is not in the Coq code and adding it does what @SkySkimmer is asking.